src/elements/paginator/compact-paginator/compact-paginator.ts

```diff
@@ -13,20 +12,17 @@ import style from './compact-paginator.scss?lit&inline';
  *
  * @event {CustomEvent<SbbPaginatorPageEventDetails>} page - Emits when the pageIndex changes.
  */
+export
 @customElement('sbb-compact-paginator')
-export class SbbCompactPaginatorElement extends SbbPaginatorCommonElementMixin(LitElement) {
+class SbbCompactPaginatorElement extends SbbPaginatorCommonElementMixin(LitElement) {
   public static override styles: CSSResultGroup = style;
   public static readonly events: Record<string, string> = {
     page: 'page',
   } as const;
 
-  protected currentPageLabel(): string {
-    return i18nPageOnTotal(this.pageIndex + 1, this.numberOfPages())[this.language.current];
-  }
-
   private _renderPageNumbers(): TemplateResult {
     return html`
-      <span class="sbb-paginator__pages" aria-label=${this.currentPageLabel()} role="group"
+      <span class="sbb-paginator__pages" role="group"
```

Comment:
Should not be needed anymore?
```suggestion
      <span class="sbb-paginator__pages"
```